src/docs/usage/reduceItemByOne().mdx

```diff
@@ -2,34 +2,16 @@
 title: reduceItemByOne()
 ---
 
-To remove an item from the cart, use `reduceItemByOne()`
+<div>
+
+`reduceItemByOne()` removes a product from the [cartItems](cartItems) array.
 
 ```jsx
 /**@jsx jsx */
```

Comment:
```suggestion
```

Reply:
Again, the real answer here might be avoiding theme-ui altogether. If you also think that's the right move I'll get started right away